tests/Bootstrap/ListGroupTest.elm

```diff
@@ -0,0 +1,210 @@
+module Bootstrap.ListGroupTest exposing (..)
+
+import Bootstrap.ListGroup as ListGroup
+import Html exposing (text)
+import Html.Attributes as Attr exposing (href)
+import Test exposing (Test, test, describe)
+import Expect
+import Test.Html.Query as Query
+import Test.Html.Selector as Selector exposing (tag, class, classes, attribute, disabled)
+
+
+{-| @ltignore
+-}
+all : Test
+all =
+    Test.concat
+        [ vanillaListGroup
+        , keyedListGroup
+        , contextualListGroup
+        , customListGroup
+        , contextualListGroup
+        ]
+
+
+vanillaListGroup : Test
+vanillaListGroup =
+    let
+        html =
+            ListGroup.ul
+                [ ListGroup.li [] [ text "List item 1" ]
+                ]
+
+        active =
+            ListGroup.ul
+                [ ListGroup.li [ ListGroup.active ] [ text "List item 1" ]
+                ]
+
+        disabled =
+            ListGroup.ul
+                [ ListGroup.li [ ListGroup.disabled ] [ text "List item 1" ]
+                ]
+    in
+        describe "vanilla ListGroup"
+            [ test "expect ul with list-group class" <|
+                \() ->
+                    html
+                        |> Query.fromHtml
+                        |> Query.has [ tag "ul", class "list-group" ]
+            , test "expect li with list-group-item class" <|
+                \() ->
+                    html
+                        |> Query.fromHtml
+                        |> Query.has [ tag "li", class "list-group-item" ]
+            , test "expect li is not disabled" <|
+                \() ->
+                    html
+                        |> Query.fromHtml
+                        |> Query.has [ tag "li", Selector.disabled False ]
```

Comment:
This assertion strikes me as a bit odd. I can sort of understand it, but if we are to test everything that should not be set... there's a lot you need to assert.